app/views/idv/in_person/ready_to_verify/show.html.erb

```diff
@@ -228,6 +228,23 @@
   </section>
 <% end %>
 
+
+<% if !@is_enhanced_ipp %>
```

Comment:
[Ruby style guidelines vote](https://rubystyle.guide/#unless-for-negatives) for `unless` over `if !`, but I know [unless is controversial](https://gsa-tts.slack.com/archives/C0NGESUN5/p1731711249352069?thread_ts=1731710722.094409&cid=C0NGESUN5).  It be would nice if we could change this to `if @is_id_ipp`, but that would require introducing a new instance variable.  I don't think the change is worth that.

What are your thoughts on `if !` versus `unless`?

Reply:
Ooooh I was not hip to the `unless` hatred! 

> kill unless with fire 
😱 

I agree if @is_id_ipp would be the most readable but also agree it isn't particularly worth the squeeze here. I don't have strong feelings about `unless` vs `if !` but I stuck with `if !` here because it was being used that way earlier in the same file. I kind of like the idiomatic feel of `unless` but now I wonder if I should avoid it in the future so as not to upset reviewers 😛